2.6/test/run

```diff
@@ -135,15 +135,15 @@ function test_config_option() {
     test_connection ${name}
 
     # If nothing is found, grep returns 1 and test fails.
-    docker exec $(get_cid $name) bash -c "cat /etc/mongod.conf" | grep -q "$setting = $value"
+    docker exec $(get_cid $name) bash -c "cat /etc/mongod.conf" | grep -q "$setting: $value"
 
     docker stop $(get_cid ${name})
 }
 
 function run_configuration_tests() {
     echo "  Testing image configuration settings"
-    test_config_option MONGODB_NOPREALLOC noprealloc true
-    test_config_option MONGODB_SMALLFILES smallfiles true
+    test_config_option MONGODB_PREALLOC preallocDataFiles false
+    test_config_option MONGODB_SMALLFILES smallFiles true
```

Comment:
should keep the test for NOPREALLOC also since it is technically still supported.